reference/src/layout/structs-and-tuples.md

```diff
@@ -141,6 +141,22 @@ struct Zst2(Zst1, Zst0);
 # }
 ```
 
+#### Structs with 1-ZST fields
+
+For the purposes of struct layout [1-ZST] fields are ignored.
+
+For example, 
+
+```rust
+type Zst1 = ();
+struct S1(i32, Zst1);
+
+type Zst2 = [u16; 0];
+struct S2(Zst2, Zst1);
+```
+
+the layout of `S1` is the same as that of `i32` and the layout of `S2` as that of `Zst2`. 
```

Comment:
This is saying more than just "1-ZST fields are ignored".

You are also assuming here that single-field structs have layout identical to their only field. That is a separate statement, and should IMO get a separate section.

Reply:
(This comment is still open.)